### PR TITLE
use gopkg.strings.Format() instead of local utils.ToString()

### PR DESCRIPTION
**Pull request by kuritka**
There are two goals for upgrade
 - replace utils by  reusable utils 
 - I'd like to use utils.Format in main.go in context of log.Debug().Msgf(...)
However utils lives in `internal` which is not enough today 
(all exists in gopkg now, we can slowly get rid of it).

### CODE REVIEW

**Review by k0da — APPROVED** [review]

File: controllers/gslb_controller_test.go [hunk 1]
```diff
@@ -339,8 +340,8 @@ func TestGslbCreatesDNSEndpointCRForHealthyIngressHosts(t *testing.T) {
 	err = settings.client.Get(context.TODO(), settings.request.NamespacedName, dnsEndpoint)
 	require.NoError(t, err, "Failed to load DNS endpoint")
 	got := dnsEndpoint.Spec.Endpoints
-	prettyGot := utils.ToString(got)
-	prettyWant := utils.ToString(want)
+	prettyGot := str.Format(got)
```
Comment (somaritane) [inline controllers/gslb_controller_test.go:343]: Please consider renaming `Format()` function back to `ToString()` in order to resolve ambiguity.
In many languages and std libraries, format functions are universal, and provide string formatting according to provided format expression.
In this current case we are dealing with simplified conversion function without parameters, plus having its internal logic.
In this case old `ToString()` name perfectly serves the purpose.

**Review by somaritane — CHANGES_REQUESTED** [review]
@kuritka left suggestion in review comments

File: controllers/gslb_controller_test.go (hunk 1 above)
Comment (kuritka) [reply controllers/gslb_controller_test.go:343]: @somaritane , please see: [gopkg #7 ](https://github.com/AbsaOSS/gopkg/pull/7)

**Comment by kuritka** [comment]
@somaritane  amended

**Review by somaritane — APPROVED** [review]
@kuritka, lgtm!

**Pull request merged**